### PR TITLE
New Diagnostic Query Notebook - Updated the filter query so that it works closes #6513

### LINKED ISSUE #6513: [Bug] New-AdsDiagnosticAdsNoteBook cannot find any queries
### Environmental information

```
#### PowerShell version:

Name                           Value                                                                                                
----                           -----                                                                                                
PSVersion                      5.1.17134.858                                                                                        
PSEdition                      Desktop                                                                                              
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                                                              
BuildVersion                   10.0.17134.858                                                                                       
CLRVersion                     4.0.30319.42000                                                                                      
WSManStackVersion              3.0                                                                                                  
PSRemotingProtocolVersion      2.3                                                                                                  
SerializationVersion           1.1.0.1                                                                                              

#### dbatools Module version:
Name    : dbatools
Path    : C:\Users\\Documents\WindowsPowerShell\Modules\dbatools\1.0.107\dbatools.psd1
Version : 1.0.107

Name    : dbatools
Path    : C:\Users\\Documents\WindowsPowerShell\Modules\dbatools\1.0.101\dbatools.psd1
Version : 1.0.101

Name    : dbatools
Path    : C:\Users\\Documents\WindowsPowerShell\Modules\dbatools\1.0.99\dbatools.psd1
Version : 1.0.99


#### SQL Server:
Microsoft SQL Server 2014 (SP2-GDR) (KB3194714) - 12.0.5203.0 (X64)   Sep 23 2016 18:13:56   Copyright (c) Microsoft Corporation  Enterprise Edition (64-bit) on Windows NT 6.3 <X64> (Build 9600: ) (Hypervisor) 

### Report



#### Host used

- [X] powershell.exe
- [ ] ISE
- [X ] VS Code
- [ ] Other (please specify)

If anything other than powershell.exe was used, please confirm that you can duplicate the issue with powershell.exe

- [X ] Still buggy in powershell.exe

#### Errors Received
![image](https://user-images.githubusercontent.com/6729780/80961176-47913980-8e02-11ea-8a09-b08b4f492ee3.png)

```

writeErrorStream      : True
PSMessageDetails      : 
Exception             : System.Exception: No diagnostic queries available for $TargetVersion = 2014
TargetObject          : 
CategoryInfo          : NotSpecified: (:) [Write-Error], Exception
FullyQualifiedErrorId : dbatools_New-DbaDiagnosticAdsNotebook,Stop-Function
ErrorDetails          : No diagnostic queries available for $TargetVersion = 2014
InvocationInfo        : System.Management.Automation.InvocationInfo
ScriptStackTrace      : at Stop-Function, 
                        C:\Users\rob.sewell\Documents\WindowsPowerShell\Modules\dbatools\1.0.101\allcommands.ps1: line 
                        82700
                        at New-DbaDiagnosticAdsNotebook<Process>, 
                        C:\Users\l\Documents\WindowsPowerShell\Modules\dbatools\1.0.101\allcommands.ps1: line 
                        53647
                        at <ScriptBlock>, <No file>: line 1
PipelineIterationInfo : {0, 1}

```

#### Steps to Reproduce

Run this 
New-DbaDiagnosticAdsNotebook -Path c:\temp\adasdas.ipynb -TargetVersion 2014

or

New-DbaDiagnosticAdsNotebook -Path c:\temp\adasdas.ipynb -SqlInstance SQLINSTANCE

any version will break in the same manner
#### Expected Behavior

The notebook to be created

#### Actual Behavior

See error above

### PR DESCRIPTION
**Pull request by SQLDBAWithABeard**
## Type of Change

 - [X ] Bug fix (non-breaking change, fixes #6513  )
 - [ ] New feature (non-breaking change, adds functionality, fixes # )
 - [ ] Breaking change (effects multiple commands or functionality, fixes # )
 - [ ] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1)
 - [ ] Adding code coverage to existing functionality
 - [ ] Pester test is included
 - [ ] If new file reference added for test, has is been added to github.com/sqlcollaborative/appveyor-lab ?
 - [ ] Nunit test is included
 - [ ] Documentation
 - [ ] Build system
 
### Purpose
Closes #6513 

### Approach
Reverts broken change

### CODE REVIEW

**Comment by wsmelton** [comment]
Your settings in VS Code are reverting the format we have in the command.

File: functions/New-DbaDiagnosticAdsNotebook.ps1 [hunk 1]
```diff
@@ -81,7 +81,8 @@ function New-DbaDiagnosticAdsNotebook {
         if (-not $TargetVersion -and -not $SqlInstance) {
             Stop-Function -Message "At least one of $SqlInstance and $TargetVersion must be provided"
             return
-        } elseif ((-not (-not $TargetVersion)) -and -not (-not $SqlInstance)) {
+        }
+        elseif ((-not (-not $TargetVersion)) -and -not (-not $SqlInstance)) {
```
Comment (wsmelton) [inline functions/New-DbaDiagnosticAdsNotebook.ps1:85]: Revert back to original format

**Comment by SQLDBAWithABeard** [comment]
Dah, apologies 
I will fix that but have just logged out so probably tomorrow

**Comment by potatoqualitee** [comment]
merci beaucoueapp

**Pull request merged**